<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
 background-color : pink;
}</string>
      </property>
      <property name="title">
       <string>Введите числа через пробел</string>
      </property>
      <widget class="QLineEdit" name="numberString">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>661</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Количество отрицательных чисел в массиве:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>471</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Сумма модулей элементов массива, расположенных после минимального по модулю числа:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>551</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Все отрицательные элементы массива заменены их квадратами. Массив отсортирован по возрастанию:</string>
       </property>
      </widget>
      <widget class="QLabel" name="negNum">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>160</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignLeft';
  border: 1px solid gray;


background-color : white;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="sum">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>180</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignLeft';
  border: 1px solid gray;


background-color : white;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="resultArray">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>661</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignLeft';
  border: 1px solid gray;
 background-color : white;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>661</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
       </property>
       <property name="text">
        <string>Расчитать</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
